neo4j/bookmarks_test.go

```diff
@@ -197,6 +197,25 @@ func TestBookmarkManager(outer *testing.T) {
 			t.Errorf("notify hook should have been called")
 		}
 	})
+
+	outer.Run("removes bookmarks of specified databases", func(t *testing.T) {
+		bookmarkManager := neo4j.NewBookmarkManager(neo4j.BookmarkManagerConfig{
+			InitialBookmarks: map[string]neo4j.Bookmarks{
+				"db":  {"z", "cooper"},
+				"foo": {"bar", "fighters"},
+				"par": {"rot", "don my French"},
+			},
+		})
+
+		bookmarkManager.Forget("db", "par")
+
+		allBookmarks := bookmarkManager.GetAllBookmarks()
+		AssertEqualsInAnyOrder(t, allBookmarks, []string{"bar", "fighters"})
+		AssertIntEqual(t, len(bookmarkManager.GetBookmarks("db")), 0)
+		AssertEqualsInAnyOrder(t, bookmarkManager.GetBookmarks("foo"),
+			[]string{"bar", "fighters"})
+		AssertIntEqual(t, len(bookmarkManager.GetBookmarks("par")), 0)
+	})
```

Comment:
Maybe check if it doesn't break if you try to forget something you don't remember.

Reply:
I need to print a t-shirt with this sentence :D

Reply:
fixed in 08543b390283f442e50a55178c13e43adbe94650